<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Full_company</class>
 <widget class="QWidget" name="Full_company">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="gridWidget_full_company" native="true">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/full_company_dark.png);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>300</number>
      </property>
      <property name="topMargin">
       <number>200</number>
      </property>
      <property name="rightMargin">
       <number>300</number>
      </property>
      <property name="bottomMargin">
       <number>200</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_code_company">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/new/prefix1/image/rang4.png);</string>
        </property>
        <property name="text">
         <string>کد شرکت را وارد کنید</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_name_company">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>نام شرکت را وارد کنید</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_code_company">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name_company">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>lineEdit_code_company</zorder>
     <zorder>pushButton_code_company</zorder>
     <zorder>pushButton_name_company</zorder>
     <zorder>lineEdit_name_company</zorder>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_bake_company">
     <property name="text">
      <string>بازگشت</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_ok_company">
     <property name="text">
      <string>تایید </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
